src/Files.App/Filesystem/FilesystemOperations/ShellFilesystemOperations.cs

```diff
@@ -17,13 +16,11 @@
 using System.Text.Json;
 using System.Threading;
 using System.Threading.Tasks;
-using Windows.ApplicationModel.AppService;
-using Windows.Foundation.Collections;
 using Windows.Storage;
 
 namespace Files.App.Filesystem
 {
-    public class ShellFilesystemOperations : IFilesystemOperations
+	public class ShellFilesystemOperations : IFilesystemOperations
```

Comment:
Forgot that small formatting issue.
```suggestion
    public class ShellFilesystemOperations : IFilesystemOperations
```